### PR TITLE
Upgraded celery to 4

### PR DESCRIPTION
**Pull request by giocalitri**
#### What are the relevant tickets?
N/A

#### What's this PR do?
Upgrades celery to 4

#### How should this be manually tested?
Try to run the mail task and verify that it runs correctly

### CODE REVIEW

File: bootcamp/celery.py [hunk 1]
```diff
@@ -16,19 +16,27 @@
 
 log = logging.getLogger(__name__)
 
+client = Client(**settings.RAVEN_CONFIG)
 
-class CustomCelery(Celery):
-    """Custom celery class to handle Sentry setup."""
+# register a custom filter to filter out duplicate logs
+register_logger_signal(client)
```
Comment (blarghmatey) [inline bootcamp/celery.py]: Is there a reason for calling this in two places?

File: bootcamp/celery.py [hunk 2]
```diff
@@ -16,19 +16,27 @@
...
-        register_logger_signal(client)
-        register_signal(client)
+# The register_logger_signal function can also take an optional argument
+# `loglevel` which is the level used for the handler created.
+# Defaults to `logging.ERROR`
+register_logger_signal(client, loglevel=logging.INFO)
 
-async = CustomCelery('bootcamp')
+# hook into the Celery error handler
+register_signal(client)
```
Comment (blarghmatey) [inline bootcamp/celery.py]: Is there a reason for calling this twice?

File: bootcamp/celery.py (hunk 1 above)
Comment (giocalitri) [inline bootcamp/celery.py]: No, I copied the configuration from Sentry docs.

File: bootcamp/celery.py (hunk 2 above)
Comment (giocalitri) [inline bootcamp/celery.py]: same as before

File: bootcamp/celery.py (hunk 1 above)
Comment (giocalitri) [inline bootcamp/celery.py]: The docs I was mentioning are here:
https://docs.sentry.io/clients/python/integrations/celery/

What I do not understand is how in this way the Sentry client is hooked to the Celery app.
In the other way it is much clearer, but I was getting this pylint error:
```
____________________________________________________________________________________________ [pylint] bootcamp/celery.py ____________________________________________________________________________________________
E: 23, 4: An attribute defined in celery.app.base line 296 hides this method (method-hidden)
```

Comment (giocalitri) [inline bootcamp/celery.py]: Ok... I verified that everything works fine with this configuration

**Comment by blarghmatey** [comment]
:+1:

**Pull request merged**